pom.xml

```diff
@@ -123,6 +123,16 @@
       <version>${javax-inject.version}</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>software.amazon.awssdk</groupId>
+      <artifactId>sso</artifactId>
+      <version>${aws-java-sdk.version}</version>
```

Comment:
these are artifacts are defined in the `bom`, so the version number can be removed